realm/realm-library/src/objectServer/java/io/realm/SyncConfiguration.java

```diff
@@ -200,29 +263,33 @@ static RealmConfiguration forRecovery(String canonicalPath, @Nullable byte[] enc
         return new RealmConfiguration(null,null, canonicalPath,null, encryptionKey, 0,null, false, OsRealmConfig.Durability.FULL, schemaMediator, null, null, true, null, true, Long.MAX_VALUE);
     }
 
-    static URI resolveServerUrl(URI serverUrl, String userIdentifier) {
+    static URI resolveServerUrl(URI baseUrl, RealmUser user) {
+        // Configure the full path to the Realm
+        String path = String.format("/api/client/v2.0/app/%s/realm-sync", user.getApp().getConfiguration().getAppId());
```

Comment:
I think all these endpoints (+ Stitch) should be in the dependencies file (built via `BuildConfig`)

Reply:
This is being handled by ObjectStore now. So this method can actually be deleted.